litellm/tests/test_api_key_override_dynamic_key.py

```diff
@@ -0,0 +1,23 @@
+from litellm import completion
+
+model = "perplexity/llama-3-sonar-large-32k-online"
```

Comment:
hi @Raymond1415926 this isn't a test which will work with pytest

you need to put it in a function like `def test_*name_of_test*()`

Reply:
also a better place to put this would be in test_completion

see example test - https://github.com/BerriAI/litellm/blob/9440754e48546912d045db205c1eefd3792700df/litellm/tests/test_completion.py#L1351